features/svc-catalog_asb/svc-catalog.feature

```diff
@@ -334,7 +333,8 @@ Feature: Service-catalog related scenarios
 
     # Delete servicebinding
     Given admin ensures "ups-binding" servicebinding is deleted
-    And I wait for the resource "secret" named "my-secret" to disappear within 60 seconds  
+    And I wait for the resource "secret" named "my-secret" to disappear within 60 seconds
+    Given admin ensures "ups-broker" clusterservicebroker is deleted after scenario  
```

Comment:
The clean up register step have to be executed before create ups-broker resource, that because the ups-broker cannot be cleaned up if test running failed before this step. In another word, move L337 to L281.

Reply:
Done